spec/unit/provider/windows_task_spec.rb

```diff
@@ -330,6 +330,9 @@
         new_resource.frequency_modifier 15
         new_resource.user "SYSTEM"
         new_resource.execution_time_limit "PT72H"
+        new_resource.start_day "3/30/2017"
+        new_resource.start_time "13:12"
+        date_time_obj = DateTime.strptime("#{new_resource.start_day} #{new_resource.start_time}", "%m/%d/%Y %H:%M")
```

Comment:
Again, this spec appears to deal with English (US) systems, which may be fine for our automated testing but is not guaranteed to be the format the user provides in the resource